### ISSUE TITLE
The save format for the Windows Store / XBox version seems to have changed somewhat and is only partially loaded

### ISSUE THREAD

**Issue by SirCathal** [goatfungus/NMSSaveEditor#974]
I play No Man Sky on GOG, Stam and via the Windows Store or on the XBox. 
The store version is very important for me, as it is the only easy way to mod the savegame on the Xbox, as it is synchronized with the Windows Store version.

Before the Omega Expedition, editing the Windows Store saves worked perfectly.

Yesterday I started playing again with the new content update and wanted to change the type of my freighter to Pirate.

When I started the editor, the current slot was displayed as empty. The log says that the game mode cannot be read. 

I have attached a few screenshots with the output and also one from the game as it should look like.

I would be very happy if you could fix this.

Console Output of the Editor:
![grafik](https://github.com/goatfungus/NMSSaveEditor/assets/69156692/c0cf7bea-3d70-40c5-aca2-ecc2c93fde2e)

This is how the Editor shows the saved games. the Date of the 3 other saves is much to old but they are shown. My Mainsave is shown as empty.
![grafik](https://github.com/goatfungus/NMSSaveEditor/assets/69156692/d4316065-366b-4eb6-8208-e908a30cc67c)

And this is how the saves are shown in the Game:
![grafik](https://github.com/goatfungus/NMSSaveEditor/assets/69156692/6740e46f-5a50-49ea-bc15-b8163add6a3b)

**Comment by goatfungus** [comment]
Can you please email me a copy of your save files?
If you could zip them all up with WinZip or something equivalent I will have a look and see what the issue is. I don't have the Xbox version of NMS so I have to rely on users to send me the save data.

**Comment by SirCathal** [comment]
Yes, you should have my email now. 
I did send it to your gmail address.
the attached 7zip File contains the complete wgs folder

**Comment by goatfungus** [comment]
Thanks for the info, I have been working on it all day and have released a new version with updated WGS/XBox support. Hope it works out for you :)

**Comment by SirCathal** [comment]
Hi,

thanks for the quick response.
I have tried the new version and it can now read the saves again.

But there is still a problem with saving.
If I have edited something, for example slot 1 manual save,
then I can read it again in the editor, but in the game no data is displayed for the manual save and only the autosave is loaded.

**Comment by goatfungus** [comment]
Yeah, there was an issue with the meta data file, but i think I have sorted it now. Let me know how it goes.

**Comment by SirCathal** [comment]
I have tested with the new version, the previous problem is now "different"

When I edit a savegame, it is now displayed in the game again. Also with the correct timestamp at which I saved (edited) it.

However, when I then load the game, the changes do not take effect.

When I start the game, it tells me that the local saves are different from those in the cloud and I can decide which one I want to use.

Of course, I then select the local saves that I have edited.
But then the ones from the cloud are actually loaded without the changes.

I then disconnected the network as a test. Then, of course, the request doesn't appear, because only the local saves can be loaded.

However, that doesn't seem to work. The game does not display an error message, but I get a fresh game as if I were starting a new game.

In the options I can see and load both the local manual save and the autosave. But when I do this I get a fresh game again
